oneandonehosting.php

```diff
@@ -56,21 +59,10 @@ public function install()
 
 	public function hookBackOfficeHeader()
 	{
-		if (strcmp(Tools::getValue('configure'), $this->name) === 0)
-		{
-			if (version_compare(_PS_VERSION_, '1.5', '>') == true)
-			{
-				$this->context->controller->addCSS($this->_path.'views/css/configure.css');
-
-				if (version_compare(_PS_VERSION_, '1.6', '<') == true)
-					$this->context->controller->addCSS($this->_path.'views/css/configure-nobootstrap.css');
-			}
-			else
-			{
-				$html = '<link rel="stylesheet" href="'.$this->_path.'views/css/configure.css" type="text/css" />';
-				$html .= '<link rel="stylesheet" href="'.$this->_path.'views/css/configure-nobootstrap.css" type="text/css" />';
```

Comment:
Hmm ... I wrote this part for PrestaShop 1.4. Even with the backward compatibility included, using `addCSS()` and `addJS()` doesn't work. Did you test it ?